resources/js/components/Concept/CreateForm.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div>
+    <BFormGroup
+      id="preferred-term-fieldset"
+      label="Preferred Term"
+      label-for="preferred-term"
+      label-size="lg"
+      label-class="font-weight-bold"
+      :invalid-feedback="preferredTermInvalid"
+      :state="preferredTermState"
+    >
+      <BFormInput
+        type="text"
+        id="preferred-term"
+        v-model="preferredTerm"
+        placeholder="Enter Preferred Term"
+        :state="preferredTermState"
+      ></BFormInput>
+    </BFormGroup>
+
+    <BFormGroup
+      id="category-fieldset"
+      label="Concept Category"
+      label-for="concept-category"
+      label-size="lg"
+      label-class="font-weight-bold"
+      :invalid-feedback="conceptCategoryInvalid"
+      :state="conceptCategoryState"
+    >
+      <BFormSelect
+        id="concept-category"
+        v-model="categoryId"
+        :options="categories"
+        :state="conceptCategoryState"
+      ></BFormSelect>
+    </BFormGroup>
+    <br />
+    <BButton
+      v-show="!saved"
+      @click="createConcept"
+      :disabled="!canSave"
+      variant="info"
+      title="Make Preferred"
+      ><i class="fa fa-floppy-o"></i> Save</BButton
+    >
+    <BButton>Add Term</BButton>
+    <div v-show="saved" class="mt-3">
+      <BButton variant="success" @click="redirectToConcept"
+        ><i class="fa fa-plus"></i> Manage Concept</BButton
+      >
+    </div>
+  </div>
+</template>
+
+<script>
+import { BFormGroup, BButton, BFormInput, BFormSelect } from 'bootstrap-vue';
+
+export default {
+  data() {
+    return {
+      preferredTerm: null,
+      conceptId: null,
+      saved: false,
+      categories: [
+        { value: process.env.MIX_ETHNICITY_ID, text: 'Ethnicity' },
+        { value: process.env.MIX_OCCUPATION_ID, text: 'Occupation' },
+        { value: process.env.MIX_ACTIVITY_ID, text: 'Activity' },
+        { value: process.env.MIX_SUBJECT_ID, text: 'Subject' },
+        { value: process.env.MIX_RELIGION_ID, text: 'Religion' },
+        { value: process.env.MIX_RELATION_ID, text: 'Relation' },
+      ],
```

Comment:
I've seen this array in a few places now. Seems like we should do something where this can be maintained in one place.

Reply:
yeah, part of me wants to add a route to the api for the app so that this array of options comes from the backend and can be updated. I am expecting some refactoring as we get closer to finished.